### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Godfrey-Alimony**
Add README File of the Project.
Add .hintrc File of the Project.
Add .stylelintrc.JSON File of the Project.
Add HTML File of the Project.
Add CSS File of the Project.
Add MIT File of the Project.
Add .gitignore File of the Project

### CODE REVIEW

**Review by zhadier — COMMENTED** [review]
Hi @Godfrey-Alimony , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that did not fulfill the Project requirements at all. The purpose of this project was to check how you do Code reviews. by doing code reviews on your older projects. You seem to have misunderstood and submitted a project on which you haven't done a code review yet. Kindly resubmit the Project when you've done your code review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      <nav class="topnav" id="navbar">
+  <a href="#BecomingaProgrammer" class="nav-link">Becoming a Programmer</a>
+<a href="#Learnhtml" class="nav-link">Learn html</a>
+<a href="#LearnCss" class="nav-link">Learn Learn Css</a>
+<a href="#LearnJavaScripts" class="nav-link">Learn JavaScripts</a>
+<a href="#LearnReact" class="nav-link">Learn React</a>
+</nav>
+ </header>
+      </div>
+            <div class="card">
```
Comment (Godfrey-Alimony) [inline index.html:25]: Please make sure to comment on your code for easy understanding of its function.

**Review by Godfrey-Alimony — COMMENTED** [review]
Due to your hard work on the project,
The project run very well.
it has a good Github flow.
it has no linters error
Repo is documented professionally.
But it requires slight changes. Thanks

File: index.html (hunk 1 above)
Comment (zhadier) [reply index.html:25]: - [ ] Hey @Piercel2022, Great job with this recommendation but you seem to have missed pointing out how they should fix it. It'll make your Review process much more meaningful if you give an example (a link) to help them understand what they have to do. Please do that for all your comments 🌥️ 
- [ ] Also, kindly ensure you find at least two different and unique things to Review, so we can have a better look at what you can do to improve your Code Reviews :)

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @Godfrey-Alimony 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✔️ Great use of Gh inline comment features

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          <p>Sunt in culpa qui officia deserunt mollit anim id est laborum consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco.</p>
+           </section>
+        </div>
+        </main>
+<div class="footer">
+  <h2>Footer</h2>
+</div>
+ <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+</body>
```
Comment (Godfrey-Alimony) [inline index.html:95]: Please make sure to comment on your code well example: https://www.freecodecamp.org/news/comment-out-html-code-example/

File: index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        </div>
+        <div class="card">
+         <section class="main-section" id="LearnReact">
+          <h3>Learn React</h3>
+          <code>Get React intime</code>
+          <h5>Title description, Sep 2, 2017</h5>
+          <div class="fakeimg">Image</div>
+          <p>Some text..</p>
+          <p>Sunt in culpa qui officia deserunt mollit anim id est laborum consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco.</p>
+           </section>
```
Comment (Godfrey-Alimony) [inline index.html:87]: Try to make your HTML code have one section for easy understanding and debugging example: https://www.w3schools.com/html/html_layout.asp.Thanks

**Review by Godfrey-Alimony — COMMENTED** [review]
I really appreciate your hard work on the project and you are so close to the next project.
Has good Github flow
Good documentation professional
No linters error.
Just make slight adjustment to go to the next project.Thanks

**Review by aimemalaika — APPROVED** [review]
Hi @Godfrey-Alimony, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**